README.md

```diff
@@ -18,27 +18,22 @@ This repository contains Docker configuration aimed at Moodle developers and tes
 
 ## Quick start
 
-```bash
-# Set up path to Moodle code
-export MOODLE_DOCKER_WWWROOT=/path/to/moodle/code
-# Choose a db server (Currently supported: pgsql, mariadb, mysql, mssql, oracle)
-export MOODLE_DOCKER_DB=pgsql
+1. Download or clone this repository
+2. Rename `.env.example` to `.env`
+3. Specify Moodle directory and db driver in `.env`
+4. Run `bin/create-moodle-config`. This copies the `config.php` into Moodle.
+You only have to call this command onces.
+4. Run `bin/moodle-docker-compose up -d` inside `moodle-docker` folder to create the container.
+5. You may have to call also `bin/moodle-docker-wait-for-db` if you use `oracle/mssql`.
```

Comment:
You may have to -> You should (at least to my knowledge you always want to do this if you execute these commands directly after each other)